tests/e2e/mapper/bluetooth/bluetooth_suite_test.go

```diff
@@ -0,0 +1,287 @@
+		cfg = utils.LoadConfig()
+		ctx = utils.NewTestContext(cfg)
+		t := time.Now()
+		nodeName = "edge-node-bluetooth"
+		nodeSelector = "node-" + utils.GetRandomString(3)
+
+		//Generate Cerificates for Edge and Cloud nodes copy to respective folders
+		Expect(utils.GenerateCerts()).Should(BeNil())
+		//Do the neccessary config changes in Cloud and Edge nodes
+		Expect(utils.DeploySetup(ctx, nodeName, "deployment")).Should(BeNil())
+
+		//Apply CRD for devicemodel
+		curPath := getpwd()
+		deviceModelPath := "../../../../build/crds/devices/devices_v1alpha1_devicemodel.yaml"
+		file := path.Join(curPath, deviceModelPath)
+		body, err := ioutil.ReadFile(file)
+		Expect(err).Should(BeNil())
+		client := &http.Client{}
+		BodyBuf := bytes.NewReader(body)
+		req, err := http.NewRequest(http.MethodPost, ctx.Cfg.K8SMasterForKubeEdge+CrdHandler, BodyBuf)
+		Expect(err).Should(BeNil())
+		req.Header.Set("Content-Type", "application/yaml")
+		resp, err := client.Do(req)
+		Expect(err).To(BeNil())
+		utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+		Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+
+		//Apply CRD for deviceinstance
+		curPath = getpwd()
+		deviceInstancePath := "../../../../build/crds/devices/devices_v1alpha1_device.yaml"
+		file = path.Join(curPath, deviceInstancePath)
+		body, err = ioutil.ReadFile(file)
+		Expect(err).Should(BeNil())
+		client = &http.Client{}
+		BodyBuf = bytes.NewReader(body)
+		req, err = http.NewRequest(http.MethodPost, ctx.Cfg.K8SMasterForKubeEdge+CrdHandler, BodyBuf)
+		Expect(err).Should(BeNil())
+		req.Header.Set("Content-Type", "application/yaml")
+		resp, err = client.Do(req)
+		Expect(err).To(BeNil())
+		utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+		Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+
+		//Run ./edgecontroller binary
+		Expect(utils.StartEdgeController()).Should(BeNil())
+
+		//Register the Edge Node to Master
+		err = utils.RegisterNodeToMaster(nodeName, ctx.Cfg.K8SMasterForKubeEdge+NodeHandler, nodeSelector)
+		Expect(err).Should(BeNil())
+
+		//Run ./edge_core after node registration
+		Expect(utils.StartEdgeCore()).Should(BeNil())
+
+		//Check node successfully registered or not
+		Eventually(func() string {
+			status := utils.CheckNodeReadyStatus(ctx.Cfg.K8SMasterForKubeEdge+NodeHandler, nodeName)
+			utils.Info("Node Name: %v, Node Status: %v", nodeName, status)
+			return status
+		}, "60s", "4s").Should(Equal("Running"), "Node register to the k8s master is unsuccessfull !!")
+
+		// Adding label to node
+		utils.ApplyLabelToNode(ctx.Cfg.K8SMasterForKubeEdge+NodeHandler+"/"+nodeName, "bluetooth", "true")
+
+		// Changing the config yaml of bluetooth mapper
+		t = time.Now()
+		pwd := getpwd()
+		sourceConfigPath:="./configuration/config.yaml"
+		sourcePath := path.Join(pwd, sourceConfigPath)
+		destinationConfigPath:= "../../../../device/bluetooth_mapper/configuration/config.yaml"
```

Comment:
I think it will be better to generate config than depend on a path ?
Thoughts @rohitsardesai83 ?